Dockerfile-debian.template

```
@@ -22,6 +22,7 @@ RUN set -eux; \
 	apt-get install -y --no-install-recommends \
 		libicu-dev \
 		libonig-dev \
+		liblua5.1-dev \
```

The package has a different name on debian, see https://www.mediawiki.org/wiki/LuaSandbox#Requirements and the empty search results here https://packages.debian.org/liblua5.1-dev. It seems like debian is still able to resolve it to the right package:
```
$ apt install liblua5.1-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'liblua5.1-0-dev' instead of 'liblua5.1-dev'
```
Maybe we should specify the exact version instead?
```suggestion
		liblua5.1-0-dev \
```